app/views/idv/enter_password/new.html.erb

```diff
@@ -18,7 +18,8 @@
 <%= simple_form_for(
       current_user,
       url: idv_enter_password_path,
-      html: { autocomplete: 'off', method: :put, class: 'margin-top-4' },
+      method: :put,
```

Comment:
> Now that I think of it, I should check that this was ever actually working as a `put` form, or if the `method` should be removed altogether in case it was never working.

It looks like either `method: :put` or `html: { method: :put }` should be equivalent

Source: https://github.com/rails/rails/blob/6f0d1ad/actionview/lib/action_view/helpers/form_helper.rb#L1593